data/shivamsinghydv.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Shivam Singh",
+  "type": "personal",
+  "bio": "Student learning",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SaraJaoude",
+      "icon": "FaTwitter"
+    },{
+      "name": "Github",
+      "url": "https://github.com/shivamsinghydv",
+      "icon": "FaGithub"
+    },{
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/shivamsinghydv/",
+      "icon": "FaLinkedIn"}
```

Comment:
typo in icon name please change it to `FaLinkedin`

Reply:
Thanks